tests/MavenTests/testEIC.cpp

```diff
@@ -1,12 +1,7 @@
 #include "testEIC.h"
 
 TestEIC::TestEIC() {
-    loadFile = "bin/methods/testsample_1.mzxml";
-    loadGoodSample = "bin/methods/testsample_2.mzxml";
-
-    files << "bin/methods/testsample_2.mzxml" << "bin/methods/testsample_3.mzxml";
-    files_ms2 << "bin/methods/ms2test1.mzML" << "bin/methods/ms2test2.mzML";
-
+    samples = new Samples();
```

Comment:
Could you change `Samples` to `TestSamples` or something? `new Samples();` is too close to `new mzSample();` imo.

Reply:
I can do that.